packages/clay-form/src/InputWithMultiSelect.tsx

```diff
@@ -45,6 +45,11 @@ export interface IProps extends React.HTMLAttributes<HTMLInputElement> {
 	 */
 	items: Array<string>;
 
+	/**
+	 * Value used for the hidden input's name attribute
```

Comment:
Maybe is better "Value used for every selectedItem hidden input's name attribute"